tests/integration/questionnaire/test_questionnaire_interstitial.py

```diff
@@ -24,7 +24,7 @@ def test_interstitial_page_button_text_is_continue(self):
         resp_url, resp = self.postRedirectGet(resp_url, post_data)
         content = resp.get_data(True)
 
-        self.assertTrue("Continue" in content, "Interstitial page does not contain Continue as button text")
+        self.assertRegex(content, "Continue")
```

Comment:
There is also an AssertContains I believe that avoids the need to use a regex...